obm-autoset/README.md

```diff
@@ -0,0 +1,46 @@
+#RackHD OBM Setting Auto Set Tool
+Tool to apply OBM setting for each compute node discovered by RackHD ora. Save the trouble to manually apply OBM setting node by node.
+
+##Installation and execution
+clone this repository.
+cd to the cloned directory.
+change the environment setting in obm_setting.json file.
+python ApplyOBMSetting.py -ora 192.168.129.96
+
+##Theroy of Operation
+The script will check the RMM/BMC address infomation in the catalog of ora. And then put OBM setting to ora by RackHD 2.0 API. The compute node should be discovered by RackHD ora first.

```

Comment:
The theroy doesn't talk about it requires user to input a list of BMC credential and the script will try every credential one by one until find a correct one.